services/slam/client_test.go

```diff
@@ -70,6 +79,22 @@ func TestClientWorkingService(t *testing.T) {
 		return mimeType, imSucc, nil, nil
 	}
 
+	workingSLAMService.GetPointCloudMapStreamFunc = func(ctx context.Context, name string) (func() ([]byte, error), error) {
+		reader := bytes.NewReader(pcd)
```

Comment:
Moved buffer outside the loop. This means it is not reallocated each loop which is fine because we only ever send over the bytes that were written to in that loop iteration